openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/edgefilters/ch/DownwardSearchEdgeFilter.java

```diff
@@ -20,19 +20,50 @@
 
 public class DownwardSearchEdgeFilter extends CHLevelEdgeFilter {
 	protected final BooleanEncodedValue accessEnc;
+	private boolean useCore = false;
+	private boolean useCoreTurnRestrictions = false;
```

Comment:
Is the default `useCoreTurnRestrictions=false` actually ever overridden? I would expect it to be set by the constructor in L41, but this doesn't happen despite the constructor having `useCoreTurnRestrictions` argument. 🤷‍♂️

Reply:
Done

Reply:
Did you maybe forget to push your changes, as I don't see any change?

Reply:
Did not push yet. Should be there now/